debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDdlParser.java

```diff
@@ -626,7 +626,7 @@ protected void parseCreateDefinition(Marker start, TableEditor table, boolean is
                     pkColumnNames.forEach(name -> {
                         Column c = table.columnWithName(name);
                         if (c != null && c.isOptional()) {
-                            table.addColumn(c.edit().optional(false).create());
+                            table.addColumn(c.edit().optional(false).unsetDefaultValue().create());
```

Comment:
What about the - admittedly unusual - case about a default value provided:

    CREATE TABLE data(id INT DEFAULT 1, PRIMARY KEY (id))

Shouldn't we only unset the default value if it is `null`?

Reply:
Ah, never mind, the default value actually is set later on. Just added a test for this and it passes.